thonny/plugins/micropython/pip_gui.py

```diff
@@ -86,15 +86,15 @@ def _show_instructions_about_target(self):
 
     def _show_extra_instructions(self):
         self._append_info_text("\n\n")
-        self._append_info_text("Under the hood " + "\n", ("caption", "right"))
+        self._append_info_text(tr("Under the hood") + "\n", ("caption", "right"))
         self._append_info_text(
-            "This dialog uses `pipkin`, a new command line tool for managing "
-            "MicroPython and CircuitPython packages. ",
+            tr("This dialog uses `pipkin`, a new command line tool for managing "
```

Comment:
Please remove the word "new" from this text. It has to be removed at some point, but removing it later would invalidate all translations written so far.

Reply:
makes sense